### DISCUSSION TITLE
FBAEMKit.framework/FBAEMKit' (no such file)

### DISCUSSION THREAD

**Discussion by lucapale** [capawesome-team/capacitor-firebase#418] [Q&A]
Hi, I have implemented the authenticator in my app.
Everything works fine in Android, but on iOS I keep getting this error:
Library not loaded: @rpath/FBAEMKit.framework/FBAEMKit
  .../Build/Products/Debug-iphonesimulator/FBAEMKit.framework/FBAEMKit' (no such file), '/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib/swift/FBAEMKit.framework/FBAEMKit' (no such file)

I have checked the the FBAEMKit seems to be loaded in the pods.
Do you have any idea what am I missing?
Thanks
Luca

**Comment by robingenz** [comment]
Sorry, I have no idea right now. Feel free to provide a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) using [this template](https://github.com/capawesome-team/.capacitor-app) in a public GitHub repository so I can debug the issue.

**Reply by lucapale** [reply]
I managed to create the example using your template.

https://github.com/lucapale/firecaptest

The app builds but when you run on the device it shows the error.
The only workaround I found has been to remove the pod file of Facebook sdk and import them through the swift package.
Let me know if you can spot the issue.
Thanks,
Luca

**Reply by robingenz** [reply]
Thank you, i will take a look as soon as possible.

**Comment by lucapale** [comment]
Thank you for your quick feedback.
It seems that the Facebook pod does not load correctly, but I don't know the reason.
I have tried to add the swift package for Facebook-ios-sdk to the POD and this seems to solve the problem.
If I'll manage to make the the minimal example with the error I will send you.
Thanks
Luca
